README.md

```diff
@@ -34,29 +34,81 @@ See [TODO.md](https://github.com/cockroachdb/cockroach/blob/master/TODO.md)
 
 ## Running Cockroach
 
-Don't have (a recent version > 1.2 of) Docker? Follow the [instructions for installing Docker on your host system](http://docs.docker.com/installation/). If you run into trouble below, check first that you're not running an old version.
+Getting started is most convenient using a recent version (>1.2) of [Docker](http://docs.docker.com/installation/).
 
 If you don't want to use Docker,
 * set up the dev environment (see [CONTRIBUTING.md](CONTRIBUTING.md))
 * `make build`
-* replace `docker [...] [run|build] [...]` by `./cockroach` below.
+* ignore the initial calls to `docker` below.
 
 #### Bootstrap and talk to a single node
 
+Getting a Cockroach node up and running is easy. If you have the `cockroach` binary, skip over the next shell session. Most users however will want to run the following:
+
 ```bash
-$ docker run -d -p 8080:8080 "cockroachdb/cockroach" \
```

Comment:
The fix to #545 is in line 61. You need to pass a template to `mktemp -d` on OSX.

Glad you liked the single liner - but if you're starting a node, and you want to use the cockroach-cli, then you better be in a shell on that node or you're going to have to pass the explicit location of the api endpoint around.

I'll see if there's another good place to put that line.